testgrid/config/config_test.go

```diff
@@ -26,7 +26,7 @@ import (
 
 	"gopkg.in/yaml.v2"
```

Comment:
also, here we can just switch to ghodss/yaml as well and drop gopkg.in/yaml dep?

Reply:
Yes and no.

Yes we can switch to that, but ghodss is just a wrapper for the same thing: https://github.com/ghodss/yaml/blob/master/yaml.go#L10

Reply:
ah right 👍 , forgot about that part

Reply:
Done